module/platform/Webots/data/config/webots.yaml

```diff
@@ -0,0 +1,10 @@
+# Controls the minimum log level that NUClear log will display
+log_level: DEBUG
+
+# Connection details
+server_address:
+port:
+
+# Info to tell webots who we are
+team_id:
+robot_id:
```

Comment:
Get this information from [`module::support::configuration::GlobalConfig`](https://github.com/NUbots/NUbots/blob/cmurtagh/webots_communication/module/support/configuration/GlobalConfig/data/config/GlobalConfig.yaml)

Reply:
Or more specifically, from the message it emits